scripts/src/cowidev/cmd/check.py

```diff
@@ -13,14 +13,21 @@
 VAX_URL = "https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/vaccinations/vaccinations.csv"
 TESTING_URL = "https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/testing/covid-testing-all-observations.csv"
 HOSP_URL = "https://raw.githubusercontent.com/owid/covid-19-data/master/public/data/hospitalizations/covid-hospitalizations.csv"
-FULL_URL = "https://covid.ourworldindata.org/data/owid-covid-data.csv"
+FULL_URL_CSV = "https://covid.ourworldindata.org/data/owid-covid-data.csv"
```

Comment:
Sorry to bother. I am not sure if this specific file represents the finalized new dataset mirroring the WHO dataset, but I thought I should try to help and point out that there is a large discrepancy in the total number of cases in China, if I am reading this correctly. Here is what I have graphed out for 2020 through Feb 27th, 2023:

![image](https://user-images.githubusercontent.com/13304149/222281942-1091c6b9-c2dd-47af-9e5a-b53e70e7fe56.png)

This of course ends in a magnitude of 2,023,904 total cases from 1-22-2020 to 2-27-2023.

However looking on the WHO dashboard here: https://covid19.who.int/region/wpro/country/cn

It appears that their accounting shows that there has been 99,030,129 confirmed cases between 3 January 2020 to 6:06pm CET, 28 February 2023

![image](https://user-images.githubusercontent.com/13304149/222282499-be25db7d-7ca5-42ca-875c-818b5b1c9e69.png)

This represents a discrepancy of 97,006,225 or an about 48 times difference in total cases.

I am not sure if this may also effect the following file, which appears to be an R tracking filter from what I can tell in the ReadMe:

[scripts/src/cowidev/megafile/steps/core.py](https://github.com/owid/covid-19-data/pull/2792/files#diff-7dcf31984ef61aeb95a4b5ddf58598674e783020778a969f6a13a42fc8ca3456)

file_url="https://github.com/crondonm/TrackingR/raw/main/Estimates-Database/database_7.csv"

Full disclosure, again, sorry to bother, hopefully I am being helpful here, but there is a prediction contest being put on by the University of Texas at Austin that I am taking part of. You have a large number of participants currently watching this repo. Here is the discussion in case you are interested. https://salemcenter.manifold.markets/SalemCenter/china-reaches-100000-covid-cases-by

Reply:
Hi @pwdel 

Thanks for getting in touch. I just scrolled through the latest comments on Manifold and can see how complicated this situation is. We won't give any particular opinion as to how to handle the resolution of the market, but if it helps, here's more information below.

The data published by Johns Hopkins University (JHU) currently shows a total of 2,023,904 confirmed cases in China.
The data published by the WHO shows a total of 99,030,129 confirmed cases in China.
The reasons for the discrepancy between the two sources are explained by the JHU team [here](https://github.com/CSSEGISandData/COVID-19/issues/6576).

The file you mentioned (`https://covid.ourworldindata.org/data/owid-covid-data.csv`) is our main COVID dataset that aggregates data from multiple sources. For confirmed cases, this file still relies on the JHU data and will do so until 8 March. On 8 March, we'll merge our pull request to start relying on WHO data instead, and the entire time series will be updated in this file.

I hope this helps!

Reply:
@edomt thank you so much for the fantastic work you have done. Obviously you are under no obligation to solve any other institution's problems at all and in truth I feel bad about even approaching you about this. I think your answer completely clears up the question I had though, thank you.